boards/arm/mimxrt1050_evk/mimxrt1050_evk.dts

```diff
@@ -22,12 +22,34 @@
 	};
 
 	chosen {
+#if defined(CONFIG_CODE_ITCM)
 		zephyr,flash = &itcm0;
+#elif defined(CONFIG_CODE_HYPERFLASH)
+		zephyr,flash = &hyperflash0;
+#elif defined(CONFIG_CODE_QSPI)
+		zephyr,flash = &qspi0;
+#endif
 		zephyr,sram = &dtcm0;
 		zephyr,console = &uart1;
 	};
 };
 
+&flexspi0 {
+	hyperflash0: hyperflash0 {
+		reg = <0x60000000 0x4000000>;
+	};
+
+	qspi0: qspi0 {
```

Comment:
Do you know what QSPI flash is on the board?  At some point we need to add a node for the actual SPI flash.

Reply:
Yes, ISSI IS25WP064AJBLE. This node is the actual SPI flash.

Reply:
Can you add a comment in the .dts that says that.  In the future I'll look at cleaning this up to have a node that describes the actual flash component, but its not needed right now.

Reply:
Sure, I'll do the same for the hyperflash.